pkg/scheduler/api/compatibility/compatibility_test.go

```diff
@@ -1276,10 +1283,17 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			}
 			wantExtenders = append(wantExtenders, extender.(*core.HTTPExtender))
 		}
-		for idx := range schedExtenders {
-			if !core.Equal(wantExtenders[idx], schedExtenders[idx].(*core.HTTPExtender)) {
-				t.Errorf("Expected extenders %+v, got %+v", wantExtenders[idx], schedExtenders[idx])
+		for i := range schedExtenders {
+			if !core.Equal(wantExtenders[i], schedExtenders[i].(*core.HTTPExtender)) {
+				t.Errorf("Want extender %+v, got %+v", wantExtenders[i], schedExtenders[i])
```

Comment:
add the index to the error message. Also, got before want: `"Got extender #%d %+v, want %+v"`